Weaver/Weaver/Weaver.cs

```diff
@@ -38,7 +38,24 @@ class WeaverLists
 
     class Weaver
     {
-        // UNetwork types
+        public static WeaverLists WeaveList { get; private set; }
```

Comment:
why 'WeaveList' when it's 'WeaverLists'?

Reply:
It allow us to quickly understand what we're looking at than "lists". But if you don't like it, I can revert it if you want. Or would you prefer WeaveLists?

Reply:
WeaveLists sounds less confusing. let's use that

Reply:
https://github.com/vis2k/Mirror/pull/455/commits/c66cd47e17ad570d80a82e8cd006a70cd3e148b4